conf/messages.en

```diff
@@ -335,7 +340,8 @@ error.importPriorityGoods.required=Select the type of priority goods
 error.importFreightType.required=Select the type of transport you''re using
 error.transportationType.required=Tell us what type of freight transportation is being used
 error.importHasALVS.required=Select yes if you''re using an Automatic Licence Verification System (ALVS)
-error.importFreightType.required=Select the type of transport you''re using
+error.contactNumber=Enter a phone number
+error.contactEmail=Enter an email address in the correct format, like name@example.com
```

Comment:
Will still need an additional error message for when a user inputs a phone number in an invalid format & the corresponding logic/tests to match 

(ie. Phone number must contain numbers only) 

Very small thing as well (and was part of the other story but seeing as it's quite simple you might wanna just pop it in here?) Anyway - was just to denote the phone number as optional & add the hint text from the spreadsheet 

https://docs.google.com/spreadsheets/d/16ix1ylUxW9mymYW5QWn-NhcnSgLH05pMkIw6bcTEcBQ/edit?pli=1#gid=1020502532

Reply:
I think it would be better to get this story done then do stuff from the other story separately to be honest.  I already have some logic for testing the phone number format but I can change the message to reflect this

Reply:
I've added a test in the model spec now as well

Reply:
Sure thing, it's a small thing anyway so no harm in leaving it for now. Thanks for making the other changes - test looks good & error message for phone is as you'd expect. 

Just need to have the email field mandatory and we should be good to go! Gimme a shout when it's up